lighthouse-core/report/html/renderer/i18n.js

```diff
@@ -66,7 +66,9 @@ class I18n {
    */
   formatMilliseconds(ms, granularity = 10) {
     const coarseTime = Math.round(ms / granularity) * granularity;
-    return `${this._numberFormatter.format(coarseTime)}${NBSP2}ms`;
+    return coarseTime === 0
```

Comment:
i believe `{signDisplay: 'exceptZero'}` in the numberFormatter's options would do this

Reply:
Well TIL! 😮 

Given that the [spec defines `exceptZero`](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-402.pdf) as using a `+` for positive numbers though, I don't think this is what we want to use here even if Chrome currently doesn't use the `+` sign for `en-US` if for no other reason than future confusion of the reader looking up the intent in the docs of `exceptZero`.

I think what we've got here is fine :)

Reply:
> Given that the spec defines exceptZero as using a + for positive numbers though,

hmm? i'm not seeing that interpretation in the HTML spec. (the pdf text is different)

https://ecma-international.org/ecma-402/#:~:text=Let%20signDisplay%20be%20numberFormat

![image](https://user-images.githubusercontent.com/39191/91869355-a2a93080-ec2a-11ea-95a1-0ddb4a6727ac.png)
![image](https://user-images.githubusercontent.com/39191/91869380-aa68d500-ec2a-11ea-804f-4c1de64dc40d.png)

Reply:
Right but the part I'm talking about is cutoff in the above screenshot and the right highlighted portion would be `positivePattern must contain the substring {plusSign}` which is not what we want for all *positive* numbers other than 0.

Reply:
Thank you for advice.
This time, I don't want to display the sign even if it is a positive value.